wgs.yaml

```diff
@@ -113,7 +182,7 @@ workinggroups:
     - github: james-jwu
       name: James Wu
       company: Google
-    - github: karkumar
+    - github: kramachandran
```

Comment:
nit: Lets move this change into https://github.com/kubeflow/community/pull/417#issuecomment-701721120

so we aren't making changes to other WGs in this PR.